lib/plugin.js

```diff
@@ -69,7 +69,8 @@ const SvgIcon = {
       attrs: {
         xmlns: 'http://www.w3.org/2000/svg',
         viewBox: this.viewBox,
-        class: '<%= options.elementClass %> <%= options.spriteClassPrefix %>' + sprite
+        // prefer staticClass over class – custom user classes are not applied till require is complete
```

Comment:
Actually, we don't need to use `staticClass`, It's better to move `class` outside of `attr`. It's more readable.
```js
...
attrs: {
   xmlns: 'http://www.w3.org/2000/svg',
   viewBox: this.viewBox
},
class: '<%= options.elementClass %> <%= options.spriteClassPrefix %>' + sprite
...
```

Reply:
Thanks a lot! 🙏